src/constants/commands.ts

```diff
@@ -0,0 +1,16 @@
+export const HELLO_COMMAND = {
+  name: "hello",
+  description: "Replies with hello in the channel",
+};
+
+export const GENERATE_LINK = {
+  name: "link",
+  description:
+    "Generate a link with user specific token to link with RDS backend.",
+};
+
+export const APPLY_CHANGE = {
+  name: "change",
+  description:
+    "command to be run after /link to change discord details of user.",
+};
```

Comment:
If this is specifically targets to change the discord details of user, then can we change the const name from "APPLY_CHANGE" to "CHANGE_USERDETAILS" ?